src/main/java/com/atlassian/bitbucket/jenkins/internal/model/BitbucketMirroredRepositoryDescriptor.java

```diff
@@ -0,0 +1,39 @@
+package com.atlassian.bitbucket.jenkins.internal.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import javax.annotation.Nullable;
+import java.util.List;
+import java.util.Map;
+
+import static java.util.Objects.requireNonNull;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class BitbucketMirroredRepositoryDescriptor {
```

Comment:
I **realy** do not like having `Descriptor` in the name here, it is genuinely confusing given a `descriptor` is something totally different in Jenkins. Can we rename to `BitbucketMirroredRepositoryDetails` instead? or another name if you have something different that does not contain the word `descriptor`

Reply:
I would like to be consistent with the other model object's, this object represents `RestMirroredRepositoryDescriptor` in Bitbucket